src/extension/module.js

```diff
@@ -1432,14 +1432,52 @@
     }
   }
 
+  /**
+   * Registers a plugin for re-evaluation if it should be shown or hidden,
+   * and if its button should be enabled or disabled.
+   * @private
+   * @param {Sidekick} sk The sidekick
+   * @param {Plugin} plugin The plugin configuration
+   * @param {HTMLElement} $plugin The plugin
+   * @returns {HTMLElement} The plugin or {@code null}
+   */
+  function registerPlugin(sk, plugin, $plugin) {
+    // re-evaluate plugin when status fetched
+    sk.addEventListener('statusfetched', () => {
+      if (typeof plugin.condition === 'function') {
+        if ($plugin && !plugin.condition(sk)) {
+          // plugin exists but condition now false
+          sk.remove(plugin.id);
+        } else if (!$plugin && plugin.condition(sk)) {
+          // plugin doesn't exist but condition now true
+          sk.add(plugin);
+        }
+      }
+      const isEnabled = plugin.button && plugin.button.isEnabled;
+      if (typeof isEnabled === 'function' || typeof isEnabled === 'boolean') {
+        const $button = $plugin && $plugin.querySelector(':scope button');
+        if ($button) {
+          if (typeof isEnabled === 'function' && isEnabled(sk)) {
```

Comment:
missing _boolean_ case:
```suggestion
          if ((typeof isEnabled === 'function' && isEnabled(sk)) || isEnabled) {
```

maybe a helper functions would be useful:
```
function isTrue(value, ...args) {
  return (typeof value === 'function' && value(...args)) || value;
}
```

Reply:
Good catch, but `boolean` is no longer supported in manifest v3, I only accept `function` here, so I can remove the check for boolean.

https://github.com/adobe/helix-sidekick-extension/blob/mv3/src/extension/module.js#L35